### DISCUSSION TITLE
Allow flux:column and flux:cell to align center

### DISCUSSION THREAD

**Discussion by lucasromanojf** [livewire/flux#254] [Feature Requests] [answered]
Currently it is only possible to align columns and cells contents to left and right (you can pass `align="right"` for column and `align="end"` for cell.

It would be good to also allow aligning column and cell contents to center (by allowing an `align="center"` prop).

**Comment by MimisK13** [comment]
To center the alignment, I had to publish all the components and edit the column.

I removed the flex.

```php
<?php else: ?>

<div class="in-[.group\/right-align]:justify-end">{{ $slot }}</div>

<?php endif; ?>
```

Now, I just pass it this way.

```html
<flux:table.column class="w-20 text-center!">#</flux:table.column>
```

**Reply by lucasromanojf** [reply]
Yes I published it too, but since it is a common use case, it could be core.

**Reply by MimisK13** [reply]
Yes, I agree. I don't like this way either... :/

**Reply by joshhanley** [reply]
@lucasromanojf and @MimisK13 yeah you shouldn't have to publish the component to get around it. Until we can provide a better option though, you can use this as a work around.

```blade
<flux:table.column class="*:justify-center">Email</flux:table.column>
```

Basically by adding the `*:` before the `justify-center` tells Tailwind to target the inner div and apply `justify-center` to that.

Hope this helps!

**Comment by joshhanley** [comment]
I've submitted a PR that adds `center` as an option for both table column and table cell.

I have also added support for align `start/end` on table column to match the align options for table cell so it's consistent. `left/right` will still work for table column, but if the PR gets merged then `start/center/end` will be the recommended options in the future.

**Reply by lucasromanojf** [reply]
Thanks Josh!

**Comment by joshhanley** [comment, marked as answer]
This has been added in Flux v2.0.5.
